.github/workflows/clippy.yml

```diff
@@ -0,0 +1,24 @@
+name: Code check
+
+on: [ pull_request ]
+
+jobs:
+  clippy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install clippy
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          components: clippy
+      - uses: actions-rs/clippy-check@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          args: --all --all-features
+  fmt:
```

Comment:
This cannot be merged until code in repo is reformatted.

Reply:
it is expected and done.